src/output.cpp

```diff
@@ -341,33 +341,76 @@ std::string Output::value_to_str(BPFtrace &bpftrace,
       return std::to_string(read_data<uint64_t>(value.data()) / div);
     }
     case Type::integer: {
-      auto sign = type.IsSigned();
-      switch (type.GetIntBitWidth()) {
-          // clang-format off
-        case 64:
-          if (sign)
-            return std::to_string(reduce_value<int64_t>(value, nvalues) / static_cast<int64_t>(div));
-          return std::to_string(reduce_value<uint64_t>(value, nvalues) / div);
-        case 32:
-          if (sign)
-            return std::to_string(
-                reduce_value<int32_t>(value, nvalues) / static_cast<int32_t>(div));
-          return std::to_string(reduce_value<uint32_t>(value, nvalues) / div);
-        case 16:
-          if (sign)
-            return std::to_string(
-                reduce_value<int16_t>(value, nvalues) / static_cast<int16_t>(div));
-          return std::to_string(reduce_value<uint16_t>(value, nvalues) / div);
-        case 8:
-          if (sign)
-            return std::to_string(
-                reduce_value<int8_t>(value, nvalues) / static_cast<int8_t>(div));
-          return std::to_string(reduce_value<uint8_t>(value, nvalues) / div);
-          // clang-format on
-        default:
-          LOG(BUG) << "value_to_str: Invalid int bitwidth: "
-                   << type.GetIntBitWidth() << "provided";
-          return {};
+      // Note we make the a conscious choice to only symbolize enums in
+      // map keys. The biggest concern is that it makes sense to do arith
```

Comment:
Did a 14 year old on twitter write this?
```suggestion
      // map keys. The biggest concern is that it makes sense to do arithmetic
```

Reply:
:D